apollo-router/src/context.rs

```diff
@@ -82,6 +106,16 @@ impl Context {
             });
         result.map_err(|e| e.into())
     }
+
+    /// Iterate over the entries.
+    pub fn iter(&self) -> impl Iterator<Item = RefMulti<'_, String, Value>> + '_ {
+        self.entries.iter()
+    }
+
+    /// Iterate mutably over the entries.
```

Comment:
Suggest adding some info about how this is not an upsert in the doc and the considerations around this.

Reply:
done: 5665976f87343018d2746bce56330459c8568b2d